gef-ui/server/middleware/headers/security.middleware.js

```diff
@@ -23,7 +23,7 @@ const security = (req, res, next) => {
   res.setHeader('X-Frame-Options', 'deny');
   res.setHeader('X-XSS-Protection', '1; mode=block');
   res.setHeader('X-Content-Type-Options', 'nosniff');
-  res.setHeader('Content-Security-Policy', "default-src 'none';connect-src 'self';base-uri 'self';block-all-mixed-content;font-src 'self';form-action 'self';frame-ancestors 'self';img-src 'self';object-src 'none';script-src 'self' 'unsafe-inline';script-src-attr 'self' 'unsafe-inline';style-src 'self';upgrade-insecure-requests");
+  res.setHeader('Content-Security-Policy', "default-src 'none';connect-src 'self';base-uri 'self';block-all-mixed-content;font-src 'self' data:;form-action 'self';frame-ancestors 'self';img-src 'self';object-src 'none';script-src 'self' 'unsafe-inline';script-src-attr 'self' 'unsafe-inline';style-src 'self';upgrade-insecure-requests");
```

Comment:
I don't remember the exact scripts setup with Portal and GEF, but I would double check in a dev/non-local environment that any inline scripts still work, e.g I think there is/was some inline scripts for Portal facility pages and maybe the main BSS/EWCS deal page.

In EXIP we discovered an issue after this same change in dev environment due to inline script not being executed, that is (I believe correctly) not picked up in E2E tests on a local machine or GHA environment.

Reply:
No `unsafe-inline` has been removed for DTFS. Thanks for flagging it up.